src/compiler/commandLineParser.ts

```diff
@@ -76,10 +76,11 @@ namespace ts {
                 "amd": ModuleKind.AMD,
                 "system": ModuleKind.System,
                 "umd": ModuleKind.UMD,
+                "es6": ModuleKind.ES6,

```

Comment:
Alright, so I'm just removing it as an option for the --module command line flag.